consensus/istanbul/backend/backend_test.go

```diff
@@ -184,6 +185,35 @@ func TestGetProposer(t *testing.T) {
 	}
 }
 
+func TestIsQBFTConsensus(t *testing.T) {
+	// Uses Istanbul DefaultConfig which has qbftBlock set to 0
```

Comment:
delete comment as it is incorrect - `newLegacyBlockchain` overrides the default and sets `qbftBlock` to `nil`

Reply:
Done